packages/ui-extensions/src/surfaces/checkout.ts

```diff
@@ -171,6 +171,7 @@ export type {
   AddressAutocompleteSuggestion,
   MatchedSubstring,
   AutocompleteAddress,
+  AddressFormField,
```

Comment:
- [ ]  @NabeelAhsen  made these changes
 - [ ]   @NabeelAhsen  approves these changes for the 2024-10 stable release.[Introduce additional address fields](https://github.com/Shopify/checkout-web/commit/ad985b2ab065b37071afe260d9d5bd7243e1236a)